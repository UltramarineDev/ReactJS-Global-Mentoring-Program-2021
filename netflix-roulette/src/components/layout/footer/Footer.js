import React from 'react';

import styles from './Footer.module.scss';
import NetflixRoulette from '../../shared/netflix-roulette';

const FooterComponent = () => (
  <div className={styles.footer}>
    <NetflixRoulette/>
  </div>
);

export default FooterComponent;
