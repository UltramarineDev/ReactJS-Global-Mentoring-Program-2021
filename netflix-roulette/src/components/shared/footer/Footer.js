import React from 'react';

import styles from '../../../../styles/Footer.module.scss';
import NetflixRoulette from '../NetflixRoulette/NetflixRoulette';

const Footer = () => (
  <div className={styles.footer}>
    <NetflixRoulette />
  </div>
);

export default Footer;
