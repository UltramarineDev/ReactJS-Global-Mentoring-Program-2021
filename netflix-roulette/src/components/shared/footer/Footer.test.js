import { render } from '@testing-library/react';
import React from 'react';

import Footer from './Footer';

describe('Tests for Footer', () => {
  test('renders MovieList', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment(<Footer />)).toMatchSnapshot();
  });
});
