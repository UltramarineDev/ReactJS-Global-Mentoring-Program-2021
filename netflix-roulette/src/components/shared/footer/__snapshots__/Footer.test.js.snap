// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Footer renders MovieList 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <span
      class="title"
    >
      <strong>
        netflix
      </strong>
      roulette
    </span>
  </div>
</DocumentFragment>
`;
